### ISSUE TITLE
Implement VR profile switching.

### ISSUE THREAD

**Issue by zeshinumi** [jrbudda/minecrift#69]
The key bindings always load from default profile no matter what. I created 2 profiles and set it to one of the ones I made, said profile name is selected, but the controls are not

**Comment by jrbudda** [comment]
Is this minecraft profiles? or VR profiles?

**Comment by zeshinumi** [comment]
sorry, VR profiles

**Comment by jrbudda** [comment]
I am of the opinion that those profiles don't actually do anything.

**Comment by zeshinumi** [comment]
it switches the VR controls when i switch them in game to what ever I set them to. It just loads the default one every time I first boot up mc

**Comment by Techjar** [comment]
Yeah, the game doesn't load the key bindings from any config, so they just end up being default. This is super annoying with mods which have key binding conflicts.

**Comment by Techjar** [comment]
Fixed in I-forget-what-version-but-it-was-recent

**Comment by jrbudda** [comment]
this is not fixed.

**Comment by Techjar** [comment]
What? Yes it is, I fixed it with some code I gave you.

**Comment by Techjar** [comment]
Oh, sorry, this is a different issue. I'm going to rename this to clarify it.
